src/sc/sidechainrpc.h

```diff
@@ -4,8 +4,11 @@
 #include "amount.h"
 
 #include "sc/sidechaintypes.h"
+#include "base58.h"
 
 //------------------------------------------------------------------------------------
+#define SC_RPC_OPERATION_DEFAULT_MINERS_FEE    10000
+#define SC_RPC_OPERATION_DEFAULT_EPOCH_LENGTH    100
```

Comment:
As a matter of style, I'd go for static const declarations here instead of defines. Something like
static const CAmount SC_RPC_OPERATION_DEFAULT_MINERS_FEE(10000);
static const int SC_RPC_OPERATION_DEFAULT_EPOCH_LENGTH(100);

Reply:
ok